### PR TITLE
New package: swftools-0.9.2

### PR DESCRIPTION
**Pull request by maciozo**

### CODE REVIEW

File: srcpkgs/swftools/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+maintainer="maciozo <maciozo@maciozo.com>"
+license="GPL-2.0-or-later"
+homepage="https://swftools.org"
+distfiles="http://swftools.org/${pkgname}-${version}.tar.gz https://xpdfreader-dl.s3.amazonaws.com/old/${_xpdf_name}-${_xpdf_version}.tar.gz"
+checksum="bf6891bfc6bf535a1a99a485478f7896ebacbe3bbf545ba551298080a26f01f1 b33a7d56f454c331ae50996f989e86c9166e57af97b74de28cddf3d51ac11f00"
+nocross="Uses a built binary to generate some SWF files before installation, which fails to run due to Exec format error."
+
+post_extract() {
+	mv ${pkgname}-${version}/* .
+	tar czf ${_xpdf_name}-${_xpdf_version}.tar.gz ${_xpdf_name}-${_xpdf_version}
```
Comment (ericonr) [inline srcpkgs/swftools/template:21]: This should use `bsdtar`, that way you can remove `tar` from `hostmakedepends`

File: srcpkgs/swftools/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+create_wrksrc=yes
+build_style=gnu-configure
+hostmakedepends="flex tar"
+makedepends="freetype-devel libjpeg-turbo-devel zlib-devel"
+short_desc="SWF manipulation and general utilities"
+maintainer="maciozo <maciozo@maciozo.com>"
+license="GPL-2.0-or-later"
+homepage="https://swftools.org"
+distfiles="http://swftools.org/${pkgname}-${version}.tar.gz https://xpdfreader-dl.s3.amazonaws.com/old/${_xpdf_name}-${_xpdf_version}.tar.gz"
+checksum="bf6891bfc6bf535a1a99a485478f7896ebacbe3bbf545ba551298080a26f01f1 b33a7d56f454c331ae50996f989e86c9166e57af97b74de28cddf3d51ac11f00"
```
Comment (ericonr) [inline srcpkgs/swftools/template:16]: These should be split one item per line.

**Review by ericonr — COMMENTED** [review]
Are the patches upstream / upstreamable?

**Comment by abenson** [comment]
Seemingly abandoned.

**Pull request closed without merging**